src/facade/dragonfly_connection.cc

```diff
@@ -789,4 +789,19 @@ std::string Connection::RemoteEndpointStr() const {
   return connection_str;
 }
 
+std::string Connection::RemoteEndpointAddress() const {
+  LinuxSocketBase* lsb = static_cast<LinuxSocketBase*>(socket_.get());
+  auto re = lsb->RemoteEndpoint();
+  return re.address().to_string();
+}
+
+std::string Connection::RemoteEndpointIp() const {
+  LinuxSocketBase* lsb = static_cast<LinuxSocketBase*>(socket_.get());
+  auto re = lsb->RemoteEndpoint();
+  // TODO: roman port is this the right property of re? I get wrong values
```

Comment:
I did not understand the question. It's a remote port of the client connection. I.e it's never 6380. What redis shows here?

Reply:
Ohh you are right I got confused. So I am not sure what is the meaning of port in this flow. We are opening several connections in the sync

Reply:
True. Lets send an empty string for now and see later what it breaks :)